Formula/spaceship.rb

```diff
@@ -0,0 +1,19 @@
+class Spaceship < Formula
+  desc "Zsh prompt for Astronauts"
+  homepage "https://denysdovhan.com/spaceship-prompt"
+  url "https://github.com/denysdovhan/spaceship-prompt/archive/v3.11.2.tar.gz"
+  sha256 "8a4ed8fd84378a3f6566355a06eb877cd217afc48be69dee08041fd5e46b8e2d"
+  license "MIT"
+  head "https://github.com/denysdovhan/spaceship-prompt.git"
+
+  def install
+    libexec.install "spaceship.zsh", "lib", "sections"
+    zsh_function.install_symlink libexec/"spaceship.zsh" => "prompt_spaceship_setup"
+  end
+
+  test do
+    ENV["SPACESHIP_CHAR_SYMBOL"] = "->"
```

Comment:
```suggestion
    ENV["SPACESHIP_CHAR_SYMBOL"] = "->"
    ENV.prepend_path "FPATH", opt_share
```
Might help; not sure.

Reply:
@carlocab I'll give that a try… thanks!

Reply:
@carlocab Nope, looks like that just messes everything else up! 🤦🏻‍♂️

Reply:
Yea, I guess the issue is that it's not defined at the point it's set here, which means that we just have `FPATH` pointing to `opt_share`. The new `zsh` process then inherits this environment variable which gets rid of the built-in `FPATH`.

I think the idea should be to find a way to add to the `FPATH` after `zsh` starts, but (I think) before calling `promptinit`.

Reply:
This would explain why it worked on the Intel nodes but not on ARM, btw. Zsh probably already searches `/usr/local/share/zsh/site-functions` without any additional prompting. However, on ARM it needs to be looking in `/opt/homebrew/share/zsh/site-functions`, but it doesn't know it needs to do that.

Reply:
I guess an easy way out would be to do `depends_on "zsh" => :test`, since Homebrew Zsh is configured to already have the right search paths.

Reply:
Makes sense. What a mess! 🙈

> I guess an easy way out would be to do `depends_on "zsh" => :test`, since Homebrew Zsh is configured to already have the right search paths.

@carlocab Seems like the best workaround… will give it a try 👍🏼